src/plugins/borrow-plugins/plugins/aave/BorrowEngineFactory.ts

```diff
@@ -47,6 +47,24 @@ export const makeBorrowEngineFactory = (blueprint: BorrowEngineBlueprint) => {
         }
       }
     }
+    const adjustCollateral = (tokenId: string | undefined, amount: string) => {
+      const index = instance.collaterals.findIndex(collateral => collateral.tokenId === tokenId)
+      if (index === -1) return // Silently fail and let the network sync reflect the change
```

Comment:
Consider log level output for these -1 cases, if appropriate

Reply:
The comment I made is misleading. This is actually just an assertion that the provided tokenId is valid (if it is not, it fails silently). Perhaps we should fail loudly?